src/index.js

```diff
@@ -99,22 +146,122 @@ const addTodo = (e) => {
         changeLocalStorage(edit.textContent, completed, editIndex);
       });
     });
-    trash.addEventListener('click', () => {
+    trash.addEventListener('click', (e) => {
       const task = e.target.parentElement.parentElement;
```

Comment:
- [ ] Please I see that you have implemented the local storage saving functionality and everything for the li element but I think you forget to add it to the todo list so nothing appears when the user reloads the page
![image](https://user-images.githubusercontent.com/74744787/208269672-1af0179f-150c-4318-8b7f-fab57d170843.png)